interface/weno/rxlogmanager.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Rx\Weno\Container;
+
+//ensure user has proper access
+if (!AclMain::aclCheckCore('patient', 'rx')) {
```

Comment:
This is for admin stuff, correct?
If so, then use:
```
if (!AclMain::aclCheckCore('admin', 'super')) {
```

Reply:
If I am wrong, then let me know, since then would need to move the link somewhere else in the menu structure where the provider can get to it.

Reply:
if providers need to see this then need to add an s to the patient:
```
if (!AclMain::aclCheckCore('patients', 'rx')) {
```

Reply:
Tina wanted this to be an admin function.